### PR TITLE
mobile: Remove bogus target in mobile/examples/cc/fetch_client/BUILD

### PR DESCRIPTION
**Pull request by RyanTheOptimist**
mobile: Remove bogus target in mobile/examples/cc/fetch_client/BUILD

Risk Level: N/A
Testing: N/A
Docs Changes: N/A
Release Notes: N/A

### CODE REVIEW

File: mobile/examples/cc/fetch_client/BUILD [hunk 1]
```diff
@@ -43,27 +43,3 @@ cc_binary(
         "@envoy//source/exe:process_wide_lib",
     ],
 )
-
-cc_test(
-    name = "fetch_client_test",
-    srcs = ["fetch_client_test.cc"],
```
Comment (yanavlasov) [inline mobile/examples/cc/fetch_client/BUILD:49]: Do you still need the .cc file?

Comment (RyanTheOptimist) [reply mobile/examples/cc/fetch_client/BUILD:49]: There is no .cc such file in the repo, so I think there's nothing to remove.

**Comment by RyanTheOptimist** [comment]
@yanavlasov friendly ping

**Review by yanavlasov — APPROVED** [review]

**Pull request merged**